content/kubermatic/master/tutorials_howtos/networking/cni_cluster_network/_index.en.md

```diff
@@ -78,8 +78,23 @@ The following table lists the supported operating systems and cloud providers fo
 **NOTE:**
 
 - A hyphen(-) denotes that the operating system is not supported for the given cloud provider.
-- This has been tested with `ebpf` proxy mode.
 
+{{% /expand%}}
+{{%expand "With ipvs proxy mode" %}}
+|                       | Ubuntu | CentOS | Flatcar | RHEL        |
+|-----------------------|--------|--------|---------|-------------|
+| AWS                   | ~[^1]  | x      | ~[^1]   | ~[^1]       |
+| Azure                 | ~[^1]  | x      | ~[^1]   | ~[^1], [^2] |
+| Google Cloud Platform | ~[^1]  | -      | -       | -           |
+| Openstack             | ~[^1]  | x      | ~[^1]   | ~[^1]       |
+**NOTE:**
+
+- A hyphen(-) denotes that the operating system is not supported for the given cloud provider.
+- A tilde(~) denotes a partial support
+
+[^1]: Pod can not access to <internal_node_ip>:<node_port>. issue [8767](https://github.com/kubermatic/kubermatic/issues/8767)
+[^2]: LoadBalancer service are not accessible from outside the cluster. issue [8768](https://github.com/kubermatic/kubermatic/issues/8768)
```

Comment:
To my understanding, #8768 is more a generic KKP Azure + RHEL issue, not specific to Cilium. Could you check if it is present on Canal as well? In that case, I would not list it here in CIlium-specific docs..

Reply:
sure, i am testing that

Reply:
as expected, this issue is not related to cilium but  Azure + RHEL issue setup. I have updated issue kubermatic/kubermatic#8768

Reply:
thanks!